common/async-runtime/Cargo.toml

```diff
@@ -17,5 +17,5 @@ runtime-async-std = ["dep:async-std"]
 runtime-tokio = ["dep:tokio"]
 
 [dependencies]
-async-std = { workspace = true, optional = true }
+async-std = { workspace = true, optional = true, features = ["unstable"] }
```

Comment:
Should not be needed, as the workspace definition has that.

Reply:
Doesn't build in my IDE if not there.

Reply:
What? Why?

Reply:
If I did cargo test on a specific crate only, it didn't work.

Reply:
Ah, cargo.